Loop/View Controllers/CarbEntryViewController.swift

```diff
@@ -58,26 +58,42 @@ final class CarbEntryViewController: LoopChartsTableViewController, Identifiable
         }
     }
 
+    fileprivate var lastEntryDate: Date?
```

Comment:
[nit] I got confused by the term "Entry" here (confusing with "carb _Entry_ Date").  If this is the "last time the user modified" the carbEntry, maybe consider something like `lastModifiedDate`?